interfaces-main/src/Employee.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+
+public class Employee {
+   
+    String name;
+    int age;
+    int salary;
+    ArrayList<Employee> bossof;
+
+    Employee(String name, int age, double salary, ArrayList<Employee> bossof){
+        this.name = name;
+        this.age = age;
+        this.salary = (int) salary;
+        this.bossof = bossof;
+    }
+
+
+    public Integer EldersRecursive(ArrayList<Employee> bossofList) {
+        int eldersWithLowSalary = 0;
+        if (bossof.size() > 0){
+            for (Employee employee : bossofList) {
+                if (employee.age > age && employee.salary < salary) {
+                    eldersWithLowSalary++;
+                }
+                EldersRecursive(employee.bossof);
```

Comment:
You are not adding the values to the ones in the recursion, this means your answer will always be the sum of the first level